test/generic/TotalFraction-test.jl

```diff
@@ -0,0 +1,287 @@
+@testset "Generic.Frac.constructors" begin
```

Comment:
all of these `Frac`s in these test set names should be `TotFrac`s

Reply:
Thanks. Fixed.